### PR TITLE
capacitor android run Hello World

### PR DESCRIPTION
**Pull request by javad9214**

### CODE REVIEW

File: package-lock.json
Comment (kerasus) [inline package-lock.json:1]: don't use npm install

File: quasar.config.js
Comment (kerasus) [inline quasar.config.js:1]: use .env ->
linuxAndroidStudio: process.env.LINUX_ANDROID_STUDIO

in .env ->
LINUX_ANDROID_STUDIO=/home/alaa/Downloads/android-studio-2022.2.1.18-linux/android-studio/bin/studio.sh

**Review by kerasus — CHANGES_REQUESTED** [review]

File: .env.example [hunk 1]
```diff
@@ -11,3 +11,5 @@ ALAA_MINIO=/minio
 NGINX_PORT=8081
 SSR_PORT=3000
 APP_ENV=development
+LINUX_ANDROID_STUDIO=/home/alaa/Downloads/android-studio-2022.2.1.18-linux/android-studio/bin/studio.sh
+APP_ID=ir.sanatisharif.android.konkur96
```
Comment (kerasus) [inline .env.example:15]: rename to ANDROID_APPLICATION_ID

File: src-capacitor/android/app/build.gradle
Comment (kerasus) [inline src-capacitor/android/app/build.gradle:1]: use env

File: src-capacitor/android/app/src/main/assets/capacitor.config.json
Comment (kerasus) [inline src-capacitor/android/app/src/main/assets/capacitor.config.json:1]: use env

File: src-capacitor/android/variables.gradle
Comment (kerasus) [inline src-capacitor/android/variables.gradle:1]: use env

File: src-capacitor/capacitor.config.json
Comment (kerasus) [inline src-capacitor/capacitor.config.json:1]: use env

File: src-capacitor/android/app/src/main/assets/capacitor.config.json
Comment (javad9214) [reply src-capacitor/android/app/src/main/assets/capacitor.config.json:1]: its can not be happen

File: src-capacitor/capacitor.config.json
Comment (javad9214) [reply src-capacitor/capacitor.config.json:1]: its can not be happen

Comment (kerasus) [inline src-capacitor/capacitor.config.json:1]: use env

**Review by kerasus — CHANGES_REQUESTED** [review]

**Pull request closed without merging**